web/src/components/modals/ChangePasswordModal/ChangePasswordModal.jsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import Modal from "react-modal";
+import Warning from "../../shared/Warning";
+import ChangePasswordForm from "./ChangePasswordForm";
+import { Link } from "react-router-dom";
+
+import "@src/scss/components/modals/ChangePasswordModal/ChangePasswordModal.scss";
+
+const ChangePasswordModal = ({ closeModal, isOpen }) => {
+  const [isSuccessful, setIsSuccessful] = React.useState(false);
+  const [identityServiceEnabled, setIdentityServiceEnabled] = React.useState(false);
+
+  const handleClose = () => {
+    closeModal();
+    setIsSuccessful(false);
+  }
+
+  const handleSetIsSuccessful = (val) => setIsSuccessful(val);
+
+  React.useEffect(() => {
+    const getLoginInfo = async () => {
+      try {
+        const response = await fetch(`${process.env.API_ENDPOINT}/login/info`, {
+          headers: {
+            "Content-Type": "application/json",
+          },
+          method: "GET",
+        });
+  
+        if (!response.ok) {
+          const res = await response.json();
+          if (res.error) {
+            throw new Error(`Unexpected status code ${response.status}: ${res.error}`);
+          }
+          throw new Error(`Unexpected status code ${response.status}`);
+        }
+  
+        const loginInfo = await response.json();
+        if (loginInfo?.method === "identity-service") {
+          setIdentityServiceEnabled(true);
+        }
+        setIdentityServiceEnabled(false);
```

Comment:
the way i read this, if they they're using an identity-service, this sets IdentityServiceEnabled to true and then sets it to false.

Reply:
Ah you're right, I need to return. Thanks for catching that. :)